src/Centreon/Domain/Monitoring/MonitoringService.php

```diff
@@ -243,6 +243,19 @@ public function isHostExists(int $hostId): bool
         return !is_null($this->findOneHost($hostId));
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function isServiceExists(int $hostId, int $serviceId): bool
+    {
+        return !is_null($this->findOneService($hostId, $serviceId));
+    }
+
+    public function findServiceGroupsByHostAndService(int $hostId, int $serviceId): ?array
```

Comment:
should be declared in the interface.
And use inheritDoc

Reply:
thanks applied.